src/test/java/edu/pdx/cs/ClientTest.java

```diff
@@ -0,0 +1,54 @@
+package edu.pdx.cs;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import static org.hamcrest.CoreMatchers.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import java.io.IOException;
+import org.mockftpserver.fake.FakeFtpServer;
+
+
+// This should probably be refactored into a different name
```

Comment:
If you find refactoring outside the scope of the current issue/story/scope of work, you can make a new issue/story/scope of work encompassing the refactoring. Comments like these tend to rot and cause confusion later. I recommend deleting.